src/sass/modules/track-a-report.scss

```diff
@@ -49,11 +49,19 @@
     }
 
     .copyright-information {
-        color: $white;
         position: absolute;
         bottom: 0;
+        right: 5px;
         margin-bottom: 0;
         font-size: x-small;
+
+      @media #{$mobile} {
+        font-size: 0.4rem;      }
+    }
+
+    .copyright-information > p {
+      color: black;
+      margin-bottom: inherit;
     }
```

Comment:
As we are using SASS this can be nested within the .copyright-information selector